conf/cb-event-forwarder.example.ini

```diff
@@ -87,6 +87,15 @@ cb_server_hostname=
 #
 # api_token=
 
+#
+#remove_from_output=highlights_by_doc,stuff
+#
+#
+#comma delimited keys will be removed from LEEF/JSON output
```

Comment:
Add some better docs here: start with a short description of what the option does, then give an example with the effects clearly stated.

```
# The remove_from_output option will remove all of the keys specified from the LEEF or JSON output.
# The list of keys are comma separated. This is useful for eliminating data that is unnecessary or 
# duplicative in your use case.
#
# for example:
#   remove_from_output=highlights_by_doc,stuff
# will remove any data with the keys "highlights_by_doc" or "stuff" from the output of the event forwarder.
```